cmd/thanos/receive.go

```diff
@@ -132,15 +130,18 @@ func registerReceive(app *extkingpin.App) {
 
 		// Local is empty, so try to generate a local endpoint
 		// based on the hostname and the listening port.
-		if *localEndpoint == "" {
-			hostname, err := os.Hostname()
-			if hostname == "" || err != nil {
-				return errors.New("--receive.local-endpoint is empty and host could not be determined.")
-			}
-			parts := strings.Split(*grpcBindAddr, ":")
-			port := parts[len(parts)-1]
-			*localEndpoint = net.JoinHostPort(hostname, port)
-		}
+		// if *localEndpoint == "" {
```

Comment:
Do you want me to uncomment this part? However, we might not be able to support the distributor mode, as the `localEndpoint` would always have a non-nil value.

Reply:
We should remove this if it's not used.

Reply:
Removed. This is a breaking change, so we should mention this in the changelog

Reply:
**Yup, this is a breaking change, so lets make a note for this**